src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,48 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  handleSelect: (tab: Tab) => void;
+};
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  handleSelect,
+}) => {
+  return (
+    <div data-cy="TabsComponent">
```

Comment:
you should use React Fragment when you have blocks on the same level. You added an additional div block to the layout